0-bootstrap/sa.tf

```diff
@@ -175,6 +175,31 @@ module "cicd_project_iam_member" {
   roles       = each.value
 }
 
+module "seed_project_remove_editor" {
```

Comment:
It would be good to add a comment as a rationale.

Reply:
Comment added